lib/formats/sh.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PROGRESS_CURR=0
-PROGRESS_TOTAL=0                           
+PROGRESS_TOTAL=0
```

Comment:
Those spaces here are (embarrassingly) needed, since the remainder after the ``=`` is replaced with the actual number. Having those spaces in allows us to do this in-place without shifting the rest of the file.